src/main/resources/default/taglib/t/actions.html.pasta

```diff
@@ -0,0 +1,119 @@
+                                           class="@action.getColorClass() @action.getIcon()"></i>
+                                    </a>
+                                </div>
+                                <div class="flex-grow-1">
+                                    <div>
+                                        <a href="@action.getUrl()" class="font-weight-bold label-js">
+                                            @action.getLabel()
+                                        </a>
+                                    </div>
+                                    <div class="text-decoration-none text-small text-muted description-js">
+                                        @action.getDescription()
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </i:for>
+                </div>
+            </div>
+        </div>
+    </i:for>
+</div>
+
+<script type="text/javascript">
+    sirius.ready(function () {
+        const _queryField = document.getElementById('actionsQueryField');
+
+        function processFilterValue() {
+            const queries = _queryField.value.toLowerCase().split(" ");
+            document.querySelectorAll('.action-js').forEach(function (_action) {
+                if (containsQueries(queries, _action)) {
+                    sirius.show(_action);
+                    _action.classList.add('action-visible-js');
+                } else {
+                    sirius.hide(_action);
+                    _action.classList.remove('action-visible-js');
+                }
+            })
+
+            document.querySelectorAll('.action-group-js').forEach(function (_group) {
+                if (_group.querySelectorAll('.action-visible-js').length === 0) {
+                    sirius.hide(_group);
+                } else {
+                    sirius.show(_group);
+                }
+            });
+        }
+
+        function containsQueries(queries, _action) {
+            let result = true;
+            queries.forEach(function (query) {
+                if (_action.querySelector('.label-js').textContent.toLowerCase().indexOf(query) < 0 &&
+                    _action.querySelector('.description-js').textContent.toLowerCase().indexOf(query) < 0) {
+                    result = false;
+                }
+            });
+
+            return result;
+        }
+
+        _queryField.addEventListener('keyup', function (event) {
+            const actions = document.querySelectorAll('.action-js');
+            if (event.keyCode === 13 && document.querySelectorAll('.action-visible-js').length === 1) {
+                // Executes the action if only one is left and ENTER is pressed in the search field...
+                document.querySelector('.action-visible-js a').click();
+                return;
+            } else if (event.keyCode === 27) {
```

Comment:
Instead of keyCode 27 (which is a hard to remember magic number) compare `event.key` to `"Escape"`